srcpkgs/removed-packages/template

```diff
@@ -166,6 +166,7 @@ replaces="
  libfcitx-qt-devel<=4.2.9.8_1
  libgksu<=2.0.12_5
  libglib-static<=2.58.3_5
+ libqmatrixclient<=0.5.3.2_1
```

Comment:
This will make people who have this package installed unable to update if XBPS is updated at the same time. I think this should be moved to a transitional dummy package, maybe.

Reply:
I don't think we need transitional dummy. Just left it there and let users remove the orphan with `xbps-remove -o`

Reply:
I still think this shouldn't be here. We don't want a repeat of https://github.com/void-linux/void-packages/issues/28779